azure-pipelines.release-vnext-nightly.yml

```diff
@@ -80,6 +82,14 @@ jobs:
           artifactName: SBom-$(System.JobAttempt)
           targetPath: $(System.DefaultWorkingDirectory)/_manifest
 
+      - task: ComponentGovernanceComponentDetection@0
+        displayName: 'Component Detection'
```

Comment:
can we improve this ? what does `Component Detection` mean ?

Reply:
Just a shortened name. I can rename it to `Component governance detection` in both pipelines on a different PR.

Reply:
that would be great, ty

Reply:
https://github.com/microsoft/fluentui/pull/23302